### PR TITLE
Fix #2119: sw600dp verions of Admin Auth Activity

### LINKED ISSUE #2119: [Tablet] Highfi AdminAuthActivity
Currently `AdminAuthActivity` screen does not have dedicated tablet layout design in develop.
Implement the `AdminAuthActivity` for landscape and portrait mode for tablet devices.
Mock Link: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/f2b8acef-b71a-40c3-a4f4-e426e6be563f/

Reference: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by Arjupta**
## Explanation
Fixes #2119
This PR adds the sw600dp versions of AdminAuthActivity as per the mocks
mentioned the issue  - 
[https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/f2b8acef-b71a-40c3-a4f4-e426e6be563f/](https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/f2b8acef-b71a-40c3-a4f4-e426e6be563f/)

Screenshots -
Screenshot_20201121-142640
Screenshot_20201123-121947

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Arjupta** [comment]
@MohamedMedhat1998 PTAL , I think @rt4914 might be unavailable right now.

**Review by MohamedMedhat1998 — CHANGES_REQUESTED** [review]
> Screenshot_20201121-142417

The screen doesn't look pretty on portrait mode.  
I suggest using guidelines with percentages instead of using fixed padding/margin or width.

**Comment by rt4914** [comment]
@ARJUPTA @MohamedMedhat1998 

I suggest instead of using guidelines and percentage. Use `minWidth=360dp` for the parent.
Reasons:
1. We are not targeting LDPI devices for now. If in future we plan to do so, we can just define the constant in dimens and control different value for LDPI and all other dimensions.
2. In these types of UI the content is very important as compared to the external padding. Therefore we should focus on the content and not the external padding (which can be variable as per device).

File: app/src/main/res/layout-sw600dp-land/admin_auth_activity.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      <ScrollView
+        android:id="@+id/admin_auth_landscape_scrollView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:overScrollMode="never"
+        android:scrollbars="none">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/admin_auth_activity.xml:59]: Similar to `port` code changes should be done in this file too. Like `minWidth="360dp"`

File: app/src/main/res/layout-sw600dp-land/admin_auth_activity.xml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+          android:paddingBottom="92dp">
+
+          <TextView
+            android:id="@+id/admin_auth_heading_textview"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="332dp"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/admin_auth_activity.xml]: Remove these start/end margins after above change.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ARJUPTA PTAL

File: app/src/main/res/layout-sw600dp-land/admin_auth_activity.xml (hunk 1 above)
Comment (Arjupta) [reply app/src/main/res/layout-sw600dp-land/admin_auth_activity.xml:59]: okay

File: app/src/main/res/layout-sw600dp-land/admin_auth_activity.xml (hunk 2 above)
Comment (Arjupta) [reply app/src/main/res/layout-sw600dp-land/admin_auth_activity.xml]: okay

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Review by MohamedMedhat1998 — APPROVED** [review]
LGTM, Thanks!

**Comment by Arjupta** [comment]
@rt4914 it can be merged now

**Comment by MohamedMedhat1998** [comment]
give @ARJUPTA 75 points

**Pull request merged**